package.json

```diff
@@ -62,7 +62,8 @@
 			"@typescript-eslint/member-ordering": "off",
 			"no-redeclare": "off",
 			"unicorn/string-content": "off",
-			"unicorn/better-regex": "off"
+			"unicorn/better-regex": "off",
+			"unicorn/prefer-number-properties": "off"
```

Comment:
?

Reply:
In `templates.js`, can we replace `parseInt` with `Number.parseInt`?

Reply:
Yes

Reply:
ok